_posts/en/newsletters/2020-08-05-newsletter.md

```diff
@@ -0,0 +1,151 @@
+  wallet firmware released for the [fee overpayment attack][].
+
+- [LND 0.11.0-beta.rc1][LND 0.11.0-beta] is the release candidate for a
+  new major release.  <!-- FIXME:harding to update Tuesday (LND haven't
+  yet published their draft release notes) -->
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #19569][] allows Bitcoin Core to fetch the parents of _orphan_
+  transactions from peers that relay transactions using wtxid. An orphan
+  transaction is an unconfirmed transaction that we receive from a peer where we
+  don't yet have the parent transactions, either as part of of our best block
+  chain, or in the mempool. More precisely, an orphan transaction has at least
+  one transaction input whose assocociated output is not in the UTXO set or our
+  mempool's outpoint map.
+
+    When we receive an orphan transaction, we place it in a temporary data
+    structure called the orphan set. We then ask the peer that sent us the
+    orphan to also send us the parent transactions that we don't yet have. We can
+    do that because the orphan refers to the txids of the parent transactions. We
+    simply send a `GETDATA` message containing those txids to the peer to
+    request the parent transactions.
+
+    For [wtxid relay peers][news108 wtxid relay], transactions are announced
+    and requested using the _wtxid_ of the transaction, not the _txid_. However,
+    orphan transactions only refer to their parents' txids, not wtxids, so it's
+    not possible to request the parent transaction using wtxid. [PR
+    #18044][Bitcoin Core #18044] (which introduced wtxid relay peers and was
+    merged last week) did not permit fetching parent transactions from wtxid peers.
+    This follow-up PR allows us to fetch those parents, by using the txid.
+
+    Fetching parent transactions using txid may eventually be replaced
+    by a [package relay][topic package relay] mechanism, where we can
+    ask a peer for all the uncofirmed ancestors of a transaction directly.
+
+- [Eclair #1491][] adds partial support for creating, using, and closing
+  channels that use [anchor outputs][topic anchor outputs] to both reduce
+  onchain transaction fees in normal cases and increase fees when
+  necessary for security.  This implementation handles all the basics
+  but does not yet support mutual channel closes or actual fee bumping;
+  those will be added later.  Additionally, interoperability testing
+  with LND's implementation revealed a case when the
+  [specification][anchor spec discuss] should be clarified.
+
+- [LND #4488][] updates the minimum CLTV expiry delta users may set to
```

Comment:
perhaps "raises the minimum CLTK expiry delta that users may set from zero or 4 blocks to a more prudent 18"